easypermissions/src/main/java/pub/devrel/easypermissions/EasyPermissions.java

```diff
@@ -207,19 +209,26 @@ public static void onRequestPermissionsResult(int requestCode, String[] permissi
 
     /**
      * If user denied permissions with the flag NEVER ASK AGAIN, open a dialog explaining the
-     * permissions rationale again and directing the user to the app settings.
+     * permissions rationale again and directing the user to the app settings. After the user
+     * returned to the app, {@link Activity#onActivityResult(int, int, Intent)} or
+     * {@link Fragment#onActivityResult(int, int, Intent)} or
+     * {@link android.app.Fragment#onActivityResult(int, int, Intent)} will be called with
+     * {@value #SETTINGS_REQ_CODE} as requestCode
      *
      * NOTE: use of this method is optional, should be called from
      * {@link PermissionCallbacks#onPermissionsDenied(int, List)}
      *
-     * @param object      the calling Activity or Fragment.
-     * @param deniedPerms the set of denied permissions.
+     * @param object                        the calling Activity or Fragment.
+     * @param deniedPerms                   the set of denied permissions.
+     * @param negativeButtonOnClickListener negative button on click listener, can be null
```

Comment:
Please make this javadoc description clearer, explaining that it's for when the user clicks the negative button in the dialog that would bring them to the settings screen.